### PR TITLE
refactor(Block: Image)

### PR DESCRIPTION
**Pull request by nikola-bleech**
Update old flynt component.

### CODE REVIEW

File: Components/BlockImage/_style.scss [hunk 1]
```diff
@@ -1,5 +1,13 @@
...
-  margin: $component-spacing-desktop 0;
+  margin: $component-spacing-mobile 0;
+
+  @media (min-width: $breakpoint-tablet) {
+    margin: $component-spacing-tablet 0;
+  }
+
+  @media (min-width: $breakpoint-desktop) {
+    margin: $component-spacing-desktop 0;
+  }
```
Comment (steffenbew) [inline Components/BlockImage/_style.scss]: Please refer to #69.

File: Components/BlockImage/functions.php [hunk 2]
```diff
@@ -14,10 +14,11 @@
                 'name' => 'image',
                 'type' => 'image',
                 'preview_size' => 'medium',
-                'instructions' => '',
+                'instructions' => 'Minimum width of the image should be 1920px.',
```
Comment (steffenbew) [inline Components/BlockImage/functions.php]: Please refer to #67.

File: Components/BlockImage/functions.php [hunk 3]
```diff
@@ -14,10 +14,11 @@
...
                 'max_size' => 4,
+                'min_width' => 1920,
                 'required' => true,
-                'mime_types' => 'gif,jpg,jpeg,png'
+                'mime_types' => 'jpg,jpeg,png'
```
Comment (steffenbew) [inline Components/BlockImage/functions.php]: Please refer to #67.

File: Components/BlockImage/_style.scss (hunk 1 above)
Comment (nikola-bleech) [reply Components/BlockImage/_style.scss]: Ignored it as per the following comment https://github.com/flyntwp/flynt/pull/69#discussion_r302961405

**Pull request merged**